app/models/post.rb

```diff
@@ -0,0 +1,5 @@
+class Post < ApplicationRecord
```

Comment:
Could you add a gem https://github.com/ctran/annotate_models ?
This way it is more simple to see which fields are now in current model.